src/modules/Reviews.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Reviews
+ */
+namespace WPML\PB\Elementor\Modules;
+
+class Reviews extends \WPML_Elementor_Module_With_Items {
+
+    protected function get_title( $field )
+    {
+        switch ( $field ) {
+            case 'content':
+                return esc_html__( 'Comment Contents', 'sitepress' );
+            case 'name':
+                return esc_html__( 'Commenter Name', 'sitepress' );
+            case 'title':
+                return esc_html__( 'Comment Title', 'sitepress' );
+            case 'image':
+                return esc_html__( 'Comment Image', 'sitepress' );
+            case 'url':
+                return esc_html__( 'Comment Link', 'sitepress' );
+            default:
+                return '';
+        }
+    }
+
+    public function get_fields()
+    {
+        return [ 'content', 'name', 'title', 'image' => [ 'url' ], 'link' => [ 'url' ] ];
+    }
+
+    protected function get_editor_type( $field )
+    {
+        if ( 'content' === $field ) {
+            return 'LINE';
```

Comment:
This should be AREA

Reply:
Thats correct.
Thank you for pointing that out to me.